config.md

```diff
@@ -231,4 +231,160 @@ If a hook returns a non-zero exit code, then an error is logged and the remainin
 `args` and `env` are optional.
 The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://golang.org/pkg/os/exec/#Cmd).
 
+## Configuration Schema Example
+
+Here is a full example `config.json` for reference.
+
+```json
+{
+	"ociVersion": "0.3.0",
+	"platform": {
+		"os": "linux",
+		"arch": "amd64"
+	},
+	"process": {
+		"terminal": true,
+		"user": {
+			"uid": 1,
+			"gid": 1,
+			"additionalGids": [5, 6]
+		},
+		"args": [
+			"sh"
+		],
+		"env": [
+			"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+			"TERM=xterm"
+		],
+		"cwd": "/",
+		"capabilities": [
+			"CAP_AUDIT_WRITE",
+			"CAP_KILL",
+			"CAP_NET_BIND_SERVICE"
+		],
+		"apparmorProfile": "",
+		"selinuxProcessLabel": ""
```

Comment:
On Tue, Mar 08, 2016 at 10:43:21AM -0800, Michael Crosby wrote:
> > +		"apparmorProfile": "",
> > +		"selinuxProcessLabel": ""
> 
> Its fine to have these in an example because how else would ppl know
> that they are there without them being in an EXAMPLE

I'm fine listing them here with meaningful content.  And regardless of
whether they're listed here, I think people should be able to discover
them via the process documentation [1] (which seems to use
‘selinuxLabel’ and not ‘selinuxProcessLabel’ anyway).

[1]: https://github.com/opencontainers/specs/blob/3b7c15d90f4becef0767b66c0d175a3ad653ff7e/config.md#process-configuration